### ISSUE TITLE
Cannot load skia library on windows7

### ISSUE THREAD

**Issue by LaoLittle** [JetBrains/skiko#546]
`skiko-windows-x64.dll` can be loaded with version 0.7.12, but not 0.7.17

![image](https://user-images.githubusercontent.com/93422095/171778118-78050212-facc-4f9b-99be-c720afb23b45.png)
: Not a valid Win32 program

```
2022-04-22 21:47:30 E/DrawMeme: java.lang.UnsatisfiedLinkError: C:\Users\Administrator\Documents\mcl\data\org.laolittle.
plugin.SkikoMirai\lib\skiko-windows-x64.dll: 找不到指定的程序。
java.lang.UnsatisfiedLinkError: C:\Users\Administrator\Documents\mcl\data\org.laolittle.plugin.SkikoMirai\lib\skiko-wind
ows-x64.dll: 找不到指定的程序。
at java.base/jdk.internal.loader.NativeLibraries.load(Native Method)
at java.base/jdk.internal.loader.NativeLibraries$NativeLibraryImpl.open(NativeLibraries.java:388)
at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:232)
at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(NativeLibraries.java:174)
at java.base/java.lang.ClassLoader.loadLibrary(ClassLoader.java:2389)
at java.base/java.lang.Runtime.load0(Runtime.java:755)
at java.base/java.lang.System.load(System.java:1953)
at org.jetbrains.skiko.Library.loadLibraryOrCopy(Library.kt:20)
at org.jetbrains.skiko.Library.findAndLoad(Library.kt:85)
at org.jetbrains.skiko.Library.load(Library.kt:59)
at org.jetbrains.skia.impl.Library$Companion.staticLoad(Library.jvm.kt:12)
```

**Comment by cssxsh** [comment]
diff 0.7.12 to 0.7.13 for windows
used GetDpiForMonitor(work in 8.1 and later versions)
[skiko/src/jvmMain/cpp/windows/drawlayer.cc](https://github.com/JetBrains/skiko/compare/v0.7.12...v0.7.13#diff-f3c56a729019345466e32e5d3cd2d151b062b9b29b0b34d945d02f49f1e8b719)

**Comment by LaoLittle** [comment]
solved in #541

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/SKIKO-546">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
